Annex60/Fluid/Sensors/Examples/FlowVelocity.mo

```diff
@@ -0,0 +1,78 @@
+within Annex60.Fluid.Sensors.Examples;
+model FlowVelocity "Test model for the flow velocity sensor"
+  import Annex60;
+  extends Modelica.Icons.Example;
+
+  package Medium = Annex60.Media.Air "Medium model";
+
+  parameter Modelica.SIunits.Area crossSection = 0.1*0.1/4*Modelica.Constants.pi
```

Comment:
I suggest to rename 'crossSection` to 'A`. Then it is consistent with the `Dampers.Exponential` and with the `Orifice` which both use `A` for the face area.